js/tools.js

```diff
@@ -706,6 +706,18 @@ function getStorageAvailable() {
 	}
 }
 
+/**
+ * Refresh bulk actions combination number after creating or deleting combinations
+ *
+ * @param sign - int
+ * @param number - int
+ */
+var refreshTotalCombinations = function (sign, number) {
```

Comment:
I don't think this is the correct file to place this function. This file is generic.

Instead, I believe this should be a method of `combinations` in `admin-dev/themes/default/js/bundle/product/product-combinations.js`